markdown/usage/troubleshooting.md

```diff
@@ -99,6 +154,101 @@ Specifying the version of the run you are using is highly recommended, as it hel
+
+For example, lets say it's the `markduplicates` process that is running out of memory (as displayed on the Nextflow running display).
+
+First we need to check to see what default memory value we have. We can do this by going to the main code of the pipeline by going to the corresponding pipeline GitHub repository and opening the `main.nf` file. You can then use your browser's find functionality for: `process markduplicates`.
+
+Once found, we then need to check the line called `label`. In this case the label could be `process_low`.
+
+Next we need to go back to the main github repository, and open `conf/base.config`. Again using our find functionality, we search for: `withLabel:'process_low'`.
+
+We see that the `memory` is set to `4.GB` with a line like: `memory = { check_max( 4.GB * task.attempt, 'memory' )})`.
+
+Now back on your local PC, we need to make a new file called `custom_resources.conf`. You should save it somewhere centrally so you can reuse it.
+
+> If you think this would be useful for multiple people in your lab/institute, we highly recommend you make an institutional profile at [nf-core/configs](https://github.com/nf-core/configs). This will simplify this process in the future.
+
+Within this file, you will need to add the following:
+
+```txt
+profiles {
+    big_data {
+      process {
+        withName: markduplicates {
+          memory = 16.GB
+        }
+      }
+    }
+}
+```
+
+Where we have increased the default `4.GB` to `16.GB`.
+
+> Note that with this you will _not_ have the automatic retry mechanism. If you want this, re-add the `check_max()` function on the `memory` line (as in the pipeline's `conf/base.config`) file, and add to the bottom of the entire file (outside the profiles block!). This block is the one starting with `def check_max(obj, type) {`, which is at the end of the [nextflow.config file](https://github.com/nf-core/eager/blob/master/nextflow.config)
+
+Once saved, we can then modify your original Nextflow run command:
+
+```bash
+nextflow run nf-core/<PIPELINE> -c /<path>/<to>/custom_resources.conf -profile big_data,<original>,<profiles> <...>
+```
+
+Where we have added `-c` to specify which file to use for the custom profiles, and then added the `big_data` profile to the original profiles you were using.
+
+:warning: it's important that big_data comes first, to ensure it overwrites any parameters set in the subsequent profiles!
+
+## Crashed pipeline with an error but Nextflow is still running
+
+If this happens, you can either wait until all other already running jobs to safely finish, or if Nextflow _still_ does not stop press `ctrl + c` on your keyboard (or equivalent) to stop the Nextflow run.
+
+> :warning: if you do this, and do not plan to fix the run make sure to delete the output folder. Otherwise you may end up a lot of large intermediate files being left! You can clean a Nextflow run of all intermediate files with `nextflow clean -f -k` or delete the `work/` directory.
```

Comment:
Output is a bit ambiguous. You mean `results` or `work`? Or both?

```suggestion
> :warning: if you do this, and do not plan to fix the run make sure to delete the output folder. Otherwise you may end up a lot of large intermediate files being left! You can clean a Nextflow run of all intermediate files with `nextflow clean -f -k` or delete the `work/` directory.
```